CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ Please follow these steps to propose a job:
 2. If your bug or enhancement matches an existing issue, please feel free to comment on that GitHub issue with your findings if you think it will help solve the issue.
 4. If there is no existing GitHub issue or Upwork job, check if the issue is happening on prod (as opposed to only happening on dev)
 5. If the issue is just in dev then it means it's a new issue and has not been deployed to production. In this case, you should try to find the offending PR and comment in the issue tied to the PR and ask the assigned users to add the `DeployBlockerCash` label. If you can't find it, follow the reporting instructions in the next item, but note that the issue is a regression only found in dev and not in prod.
-6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Include all relevant data, screenshots and examples in the post or thread.
+6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source.  
```

Comment:
@mallenexpensify To be honest, I believe this line is the best in this file. I feel like the instructions are most handy when someone wants to propose a new feature, which is mentioned on this line. So I suggest:

```suggestion
6. If the issue happens in production then report the issue(s) in the [#expensify-open-source](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions) Slack channel, prefixed with `BUG:` or `Feature Request:`. Please use the templates for bugs and feature requests that are bookmarked in #expensify-open-source.  For instructions on how to come up with a plan, which could be useful for you in this stage especially in case of a feature request, please refer to [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md).
```

Reply:
@mountiny , this shortened version will likely work better. 
> View [this guide](https://github.com/Expensify/App/blob/main/HOW_TO_CREATE_A_PLAN.md) for help creating a plan when proposing a feature request

How would I delete yours and add mine above? I'm unfamiliar with editing a PR

Reply:
@mallenexpensify I think what you can do is to click on the branch name here:
![image](https://user-images.githubusercontent.com/36083550/152010188-b16c62fc-162d-467f-ac80-a5749be839fc.png)

`mallenexpensify-patch-1` in this case. That will bring you to your version of the code which we can see in this PR. Then navigate to the Contributing file https://github.com/Expensify/App/blob/mallenexpensify-patch-1/CONTRIBUTING.md

And then you cum click the pencil button same as before, make the changes and commit it.